elements/lisk-cryptography/test/bls_lib/lib.spec.ts

```diff
@@ -60,231 +59,259 @@ interface EthFastAggrVerifySpec {
 }
 
 describe('bls_lib', () => {
-	const groupOrder = '0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001';
+	describe('isNotMultipleOfGroupOrder', () => {
+		it('should return true if sk is not equal to groupOrder or groupOrderDouble', () => {
+			const sk = Buffer.from(
+				'73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000003',
+				'hex',
+			);
+			expect(isNotMultipleOfGroupOrder(sk)).toBe(true);
+		});
 
-	describe('isSecretKeyNonZeroModOrder', () => {
-		it('should return true when given a valid secret key', () => {
-			const secretKey = SecretKey.fromBytes(Buffer.alloc(32, 1));
-			expect(isSecretKeyNonZeroModOrder(secretKey)).toBe(true);
+		it('should return true if sk is 3*groupOrder', () => {
+			const sk = Buffer.from(
+				'15bc8f5f97cd877d899ad88181ce5880ffb38ec08fffb13fcfffffffd00000003',
+				'hex',
+			);
+			expect(isNotMultipleOfGroupOrder(sk)).toBe(true);
 		});
 
-		it('should throw error when given a zero secret key', () => {
-			expect(() => isSecretKeyNonZeroModOrder(SecretKey.fromBytes(Buffer.alloc(32, 0)))).toThrow(
-				'ZERO_SECRET_KEY',
+		it('should return true when the buffer is not a multiple of the group order and 32 bytes long', () => {
```

Comment:
Also this one I would remove for the same reason as the previous one. Moreover, the description of the text doesn't fit, because this secret key is indeed a multiple of the group order (`0*r`) and is 32 bytes long.